File: OOPS/java/p3.java
import java.util.Scanner;

public class Main {

    int add(int x, int y) {
        return x + y;
    }

    int sub(int x, int y) {
        return x - y;
    }

    int mul(int x, int y) {
        return x * y;
    }

    int div(int x, int y) {
        return x / y;
    }

    public void main(String[] args) {

        Scanner oprt = new Scanner(System.in);

        System.out.println("Enter the operation you want to perform:");
        System.out.println("1. Addition");
        System.out.println("2. Subtraction");
        System.out.println("3. Multiplication");
        System.out.println("4. Division");

        int a = oprt.nextInt();

        System.out.print("Enter num1: ");
        int x = oprt.nextInt();

        System.out.print("Enter num2: ");
        int y = oprt.nextInt();

        switch (a) {
            case 1:
                System.out.println("Addition: " + add(x, y));
                break;

            case 2:
                System.out.println("Subtraction: " + sub(x, y));
                break;

            case 3:
                System.out.println("Multiplication: " + mul(x, y));
                break;

            case 4:
                if (y != 0) {
                    System.out.println("Division: " + div(x, y));
                } else {
                    System.out.println("Error: Division by zero is not allowed.");
                }
                break;

            default:
                System.out.println("Wrong input! Please enter a number between 1 and 4.");
        }

        oprt.close();
    }
}

File: OOPS/p3.cpp
#include<iostream>

int main(){

	int fun(){

		       int x,y;
		       std::cout<<"Enter your NUMBERS: \n";
		       std::cin>>x>>y;
		       return 0;
	}



	int num;
	std::cout<<"Enter your choice\n1.ADDITION\n2.SUBTRACTION\n3.MULTIPLICATION\n4.DIVISION\n5.Exit\n";
	std::cin>>x;

	switch(num):
	case 1:
		       fun();
		       std::cout<<"ADDITION OF X AND Y IS : ">>x+y>>std::endl;
		      break;
	case 2:
       		      fun();
		      std::cout<<"SUBTRACTION OF X AND Y IS : "<<x-y<<std::endl;

		      break;
	case 3:
		      fun();
		      std::cout<<"MULTIPLICATION OF X AND Y IS : "<<x*y<<std::endl;
		      break;
	case 4:
		      fun();
		      std::cout<<"DIVISION OF X AND Y IS : "<<x/y<<std::endl;
	case 5:
		     std::cout<<"ENTER THE CORRECT CHOICE";


